src/sgg_utils/foreup_utils.py

```diff
@@ -114,6 +114,30 @@ def get_teesheet(token, course_id, teesheet_id, include: list = []):
 
     return content
 
+def get_customer(token, course_id, customer_id, include=[]):
+    headers = {
+        'Content-Type': 'application/json',
+        'x-authorization': f'Bearer {token}'
+    }
+
+    if len(include) > 0:
+        included = '?include=' + '&'.join(include)
```

Comment:
The join delimiter should be "," instead of "&". I made a change yesterday to a few of the other functions that still used &.